.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,18 +12,41 @@ are the most critical to review.
 
 closes: #XXXX
 
+
+### Commit Message / Changelog entry
```

Comment:
cc @colin-axner @charleenfei  @damiannolan @crodriguezvega 

I updated the PR template to include a new section where contributors will include the proposed commit message / changelog entry. (can thumbs up if you're happy, and leave a comment if you have any suggestions / concerns! )